uwg_schema/model.py

```diff
@@ -273,15 +278,19 @@ def check_bld(cls, value):
         'urban grass.'
     )
 
-    zone: int = Field(
+    zone: str = Field(
         ...,
-        ge=1,
-        le=16,
-        description='Index representing an ASHRAE climate zone. Choose from the '
-        'following indices: 1 (1A), 2 (2A), 3 (2B), 4 (3A), 5 (3B-CA), 6 (3B), 7 (3C), '
-        '8 (4A), 9 (4B), 10 (4C), 11 (5A), 12 (5B), 13 (6A), 14 (6B), 15 (7), 16 (8).'
+        description='Text representing an ASHRAE climate zone. Choose from the '
+        'following: "1A", "2A", "2B", "3A", "3B-CA", "3B", "3C", "4A", "4B", '
+        '"4C", "5A", "5B", "6A", "6B", "7", "8".'
```

Comment:
Is this only used to set default constructions?  If so, can you make a note that this parameter doesn't affect the simulation when detailed BEMDefs are specified.

Reply:
Pretty much. It's only used if the reference DOE buildings are used to define the building stock. It effects a couple of the properties in the those reference BEMDef, constructions, some HVAC parameters as well (i.e heating efficiency). So it won't effect custom references. I can add a note.